files/Aboot/boot0.j2

```diff
@@ -42,14 +42,14 @@ parse_environment_config() {
 
 extract_image() {
     ## Remove all the other unnecssary files except swi file, boot-config
-    for f in $(ls -A $target_path); do
+    for f in $(ls -A $target_path/image-%%IMAGE_VERSION%%); do
```

Comment:
Instead of adding image-%%IMAGE_VERSION%% everywhere it could be cleaner to just define a variable and use it instead of target_path where needs be.
image_path="$target_path/image-%%IMAGE_VERSION%%"